src/Components/UpComingQuiz/UpComingQuiz.tsx

```diff
@@ -0,0 +1,77 @@
+import psyology from "../../assets/443de449f9295558ab8120f068b0b58b.png";
+import intro from "../../assets/Quiz img (2).png";
+import { Link } from "react-router-dom";
+
+export default function UpComingQuiz() {
+  return (
+    <div className="upcoming-quizzes p-3 w-full">
+      <div className="border rounded-xl">
+        <h5 className="text-lg font-semibold my-2 mx-2">Upcoming quizzes</h5>
+
+        <div className="quizez m-3 my-4">
```

Comment:
You can abstract every quiz card as a component such as `UpcomingQuiz` and reuse it here to make the list of `UpcominQuizes`